src/helpers/pivot/pivot_time_adapter.ts

```diff
@@ -183,7 +183,7 @@ const quarterNumberAdapter: PivotTimeAdapterNotNull<number> = {
   toValueAndFormat(normalizedValue) {
     return {
       value: _t("Q%(quarter_number)s", { quarter_number: normalizedValue }),
-      format: "0",
+      format: "@",
```

Comment:
should also be adapter for new granularities: day of week, hours, minutes, seconds